packages/lesswrong/lib/collections/posts/fragments.ts

```diff
@@ -66,6 +66,7 @@ registerFragment(`
     contactInfo
     isEvent
     types
+    groupId
```

Comment:
@jpaddison3 This was the issue that prevented the existing `groupId` value from populating the edit event form. This fragment has `group` in it (below), which contains the group's `_id`, but didn't have `groupId`.

Reply:
Wait really? How the heck does that work? Does the server have some sort of default event we query? I'd love to have you show me how this works at dev checkin tomorrow.

Reply:
`groupId` is a field on DB objects, `group` is a resolver which is created by `foreignKeyField` which does a join.

Reply:
I know how it resolves, but I don't know how the form resolves a value before(?) the object exists.